### PR TITLE
Add Metadata sheet

### PR DESCRIPTION
**Pull request by dafeda**
Shows which verison of fmu-tools was used and when it was created.

image

### CODE REVIEW

File: src/fmu/tools/sensitivities/create_design.py [hunk 1]
```diff
@@ -194,6 +196,18 @@ def to_xlsx(
         defaults.to_excel(
             xlsxwriter, sheet_name=defaultsheet, index=False, header=False
         )
+
+        version_info = pd.DataFrame(
+            {
+                "Description": ["Created using fmu-tools version:", "Created on:"],
+                "Value": [
+                    fmu.tools.__version__,
+                    datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
```
Comment (berland) [inline src/fmu/tools/sensitivities/create_design.py:205]: Recommend to use `datetime.now().isoformat(sep=" ", timespec="seconds")`

File: tests/sensitivities/test_create_design.py [hunk 2]
```diff
@@ -161,6 +164,24 @@ def test_generate_onebyone(tmpdir):
...
+    assert (diskmetadata.columns == ["Description", "Value"]).all()
+    assert diskmetadata["Description"].iloc[0] == "Created using fmu-tools version:"
+    assert diskmetadata["Value"].iloc[0] == fmu.tools.__version__
+    assert diskmetadata["Description"].iloc[1] == "Created on:"
+
+    # For the timestamp, we can't check the exact value since it will differ
+    # each time the test runs. Instead, we can verify it's a valid datetime string
+    # by attempting to parse it
+    timestamp = diskmetadata["Value"].iloc[1]
+    try:
```
Comment (berland) [inline tests/sensitivities/test_create_design.py:180]: assert instead that `datetime.fromisoformat()` does not err

Comment (berland) [reply tests/sensitivities/test_create_design.py:180]: You could mock datetime.now() if you want something exact to assert, but I am fine with this :)

File: src/fmu/tools/sensitivities/create_design.py (hunk 1 above)
Comment (dafeda) [reply src/fmu/tools/sensitivities/create_design.py]: Thanks, fixed.

File: tests/sensitivities/test_create_design.py (hunk 2 above)
Comment (dafeda) [reply tests/sensitivities/test_create_design.py:180]: Thanks, I'm now using `datetime.fromisoformat()` instead. Don't want to do mocking though! 😬

**Review by berland — APPROVED** [review]
👍🏻 👍🏻

**Pull request merged**